test/unit/cookie.test.js

```diff
@@ -194,4 +194,142 @@ describe('Cookie', function () {
+            expires: 1502442248,
+            hostOnly: false,
+            httpOnly: false,
+            key: '_ga',
+            path: '/',
+            secure: false,
+            session: false,
+            value: 'GA1.2.113558537.1435817423'
+        };
+
+        it('should return single Set-Cookie header string', function () {
+            var cookie = new Cookie(rawCookie);
+            expect(Cookie.stringify(cookie)).equals(Cookie.prototype.toString.call(cookie));
+        });
+    });
+
+    describe('toString', function () {
+        it('should return single Set-Cookie header string with key value', function () {
+            var rawCookie = {
+                    key: '_ga',
+                    value: 'GA1.2.113558537.1435817423'
+                },
+                cookie = new Cookie(rawCookie);
+            expect(cookie.toString())
+                .equals('_ga=GA1.2.113558537.1435817423');
+        });
+
+        it('should return single Set-Cookie header string with expires', function () {
+            var rawCookie = {
+                    key: '_ga',
+                    value: 'GA1.2.113558537.1435817423',
+                    expires: '14 Jun 2017 00:00:00 PDT'
+                },
+                cookie = new Cookie(rawCookie);
+            expect(cookie.toString())
+                .equals('_ga=GA1.2.113558537.1435817423; Expires=Wed, 14 Jun 2017 07:00:00 GMT');
+        });
+
+        it('should return single Set-Cookie header string with secure and httpOnly set to true', function () {
+            var rawCookie = {
+                    key: '_ga',
+                    value: 'GA1.2.113558537.1435817423',
+                    httpOnly: true,
+                    secure: true
+                },
+                cookie = new Cookie(rawCookie);
+            expect(cookie.toString())
+                .equals('_ga=GA1.2.113558537.1435817423; Secure; HttpOnly');
+        });
+
+        it('should return single Set-Cookie header string with maxAge value', function () {
+            var rawCookie = {
+                    key: '_ga',
+                    value: 'GA1.2.113558537.1435817423',
+                    maxAge: 1502442248
+                },
+                cookie = new Cookie(rawCookie);
+            expect(cookie.toString())
+                .equals('_ga=GA1.2.113558537.1435817423; Max-Age=1502442248');
+        });
+
+        it('should return single Set-Cookie header string with extensions', function () {
+            var rawCookie = {
+                    key: '_ga',
+                    value: 'GA1.2.113558537.1435817423',
+                    maxAge: 1502442248,
+                    extensions: [{
+                        key: 'Priority',
+                        value: 'HIGH'
+                    }]
+                },
+                cookie = new Cookie(rawCookie);
+            expect(cookie.toString())
+                .equals('_ga=GA1.2.113558537.1435817423; Max-Age=1502442248; [object Object]');
```

Comment:
What's `[object Object]`?

Reply:
with extensions as part of cookie `toString` converts to [object Object] is it valid, thus need to do this comparison